cpp/turbodbc_arrow/Test/tests/arrow_result_set_test.cpp

```diff
@@ -114,7 +116,7 @@ TEST_F(ArrowResultSetTest, SimpleSchemaConversion)
         "int_column", turbodbc::type_code::integer, size_unimportant, true}};
     EXPECT_CALL(rs, do_get_column_info()).WillRepeatedly(testing::Return(expected));
 
-    turbodbc_arrow::arrow_result_set ars(rs);
+    turbodbc_arrow::arrow_result_set ars(rs, false);
```

Comment:
Instead of using `true` and `false` in the tests, please consider creating and using constants like
```
bool const strings_as_strings = false;
bool const strings_as_dictionaries = true;
```
That should make the tests more readable.